www/includes/easyparliament/templates/html/homepage/featured-content.php

```diff
@@ -0,0 +1,22 @@
+
+<?php $featured_item = $annoucement_manager->get_random_valid_item("homepage"); ?>
+
+<?php if ( $featured_item ) { ?>
+<h2>The latest</h2>
+<div class="featured-content__wrapper">
+<a href="<?= $featured_item->url ?>"><img class="featured-content__image" src="<?= $featured_item->thumbnail_image_url ?>" alt="<?= $featured_item->thumbnail_image_alt_text ?>"></a>
+<div>
+    <a href="<?= $featured_item->url ?>"><h3 class="featured-content__title"><?= $featured_item->title ?></h3></a>
+    <p class="featured-content__description"><?= $featured_item->content ?></p>
+    <a class="button featured-content__button" href="<?= $featured_item->url ?>"><?= (isset($featured_item->button_text) ? $featured_item->button_text : "Read more") ?></a>
+</div>
+</div>
+
+<?php } else { ?>
+
+    <?php if ( count($featured) > 0 ) {
+                        include 'featured.php';
```

Comment:
This means the featured.php will be shown twice on the page if there's no homepage announcement.

Reply:
Yes, I couldn't see a good way around this because some content we want in this box above the search (the alerts link) on the right all the time. My assumption (which hopefully won't be wrong) is we'll always have something there, and if not, it won't be immediately apparent to most people landing it's duplicated above and below the search.

Reply:
We could set a global flag so if it's being included the first time and the second time checks if the flag has been set and doesn't show it again, but as you say, no biggie.

Reply:
I've done this just so it isn't a surprise later (or when we first turn it on). Looks like it works fine.